Error in building the application. Please, check ./Makefile

@fadeev 

When I finish "starport add wasm", and execute "starport serve", an exception like the title appears！

Whether to support adding "wasm"？

 thank you!

@fadeev After executing make, as follows：

fatal: No names found, cannot describe anything.
fatal: your current branch 'master' does not have any commits yet
go install -mod=readonly -ldflags '-X github.com/cosmos/cosmos-sdk/version.Name=NewApp -X github.com/cosmos/cosmos-sdk/version.ServerName=blogd -X github.com/cosmos/cosmos-sdk/version.ClientName=blogcli -X github.com/cosmos/cosmos-sdk/version.Version= -X github.com/cosmos/cosmos-sdk/version.Commit= ' ./cmd/blogd
# github.com/keybase/go-keychain
cgo-gcc-prolog:203:11: warning: 'SecTrustedApplicationCreateFromPath' is deprecated: first deprecated in macOS 10.15 - No longer supported [-Wdeprecated-declarations]
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/Security.framework/Headers/SecTrustedApplication.h:59:10: note: 'SecTrustedApplicationCreateFromPath' has been explicitly marked deprecated here
# github.com/cuiran/blog/app
app/app.go:181:33: not enough arguments in call to wasm.NewKeeper
        have (*amino.Codec, *"github.com/cosmos/cosmos-sdk/store/types".KVStoreKey, "github.com/cosmos/cosmos-sdk/x/auth/keeper".AccountKeeper, "github.com/cosmos/cosmos-sdk/x/bank/internal/keeper".Keeper, "github.com/cosmos/cosmos-sdk/x/staking/keeper".Keeper, "github.com/cosmos/cosmos-sdk/types".Router, string, "github.com/CosmWasm/wasmd/x/wasm/internal/types".WasmConfig, string, nil, nil)
        want (*amino.Codec, "github.com/cosmos/cosmos-sdk/store/types".StoreKey, subspace.Subspace, "github.com/cosmos/cosmos-sdk/x/auth/keeper".AccountKeeper, "github.com/cosmos/cosmos-sdk/x/bank/internal/keeper".Keeper, "github.com/cosmos/cosmos-sdk/x/staking/keeper".Keeper, "github.com/cosmos/cosmos-sdk/types".Router, string, "github.com/CosmWasm/wasmd/x/wasm/internal/types".WasmConfig, string, *"github.com/CosmWasm/wasmd/x/wasm/internal/keeper".MessageEncoders, *"github.com/CosmWasm/wasmd/x/wasm/internal/keeper".QueryPlugins)
make: *** [install] Error 2

hi, @fadeev 
After executing make, the following appears
cmd/blogd/main.go:85:22: undefined: store.NewPruningOptionsFromString
make: *** [install] Error 2
Thanks!

--- a/starport/templates/add/new.go
+++ b/starport/templates/add/new.go
@@ -71,7 +71,7 @@ func appModify(opts *Options) genny.RunFn {
 	}
 	wasmConfig := wasmWrap.Wasm
 	supportedFeatures := "staking"
-	app.wasmKeeper = wasm.NewKeeper(app.cdc, keys[wasm.StoreKey], app.accountKeeper, app.bankKeeper, app.stakingKeeper, wasmRouter, wasmDir, wasmConfig, supportedFeatures, nil, nil)	`
+	app.wasmKeeper = wasm.NewKeeper(app.cdc, keys[wasm.StoreKey], app.subspaces[wasm.ModuleName], app.accountKeeper, app.bankKeeper, app.stakingKeeper, wasmRouter, wasmDir, wasmConfig, supportedFeatures, nil, nil)`
 		content = strings.Replace(content, placeholder4, template4, 1)
 
 		template5 := `%[1]v

